Library/Formula/txr.rb

```diff
@@ -0,0 +1,16 @@
+class Txr < Formula
+  desc "Original, New Programming Language for Convenient Data Munging"
```

Comment:
This is a nitpick, but we use sentence-casing for this - you don't need to capitalize words past the first.